### PR TITLE
[v5]: Upgrade styled components

### PR DESCRIPTION
**Pull request by poteirard**
Please review the styles with care. As I don't know all the new designs

Closes #187 #188

### CODE REVIEW

File: src/components/atoms/Button/Button.tsx [hunk 1]
```diff
@@ -54,7 +54,9 @@ export const buttonStyle = css<IButtonProps>`
   ${({ styling = 'primary' }) => {
     const { bg } = colorMap[styling];
     const isActionGradient = bg === colors.action;
-    const bgFallback = `background-color: ${colors.actionPlain}`;
+    const bgFallback = css`
```
Comment (poteirard) [inline src/components/atoms/Button/Button.tsx:57]: This was making the button look ugly 😓 Maybe there are some other bugs but we can iterate over it.

File: src/components/organisms/Form/FormButton/FormButton.test.tsx [hunk 2]
```diff
@@ -38,8 +38,7 @@ describe('<FormButton />', () => {
 
   it('renders disabled button', async () => {
     const { getByText } = renderComponent();
-    await wait();
```
Comment (poteirard) [inline src/components/organisms/Form/FormButton/FormButton.test.tsx:41]: wait is deprecated

File: src/components/organisms/Navbar/Navbar/__snapshots__/Navbar.test.tsx.snap [hunk 3]
```diff
@@ -5,6 +5,22 @@ exports[`<Navbar /> should render component with all the props 1`] = `
...
+}
+
+@media (min-width:992px) {
+
+}
+
+@media (min-width:1300px) {
+
+}
+
```
Comment (thegrinder) [inline src/components/organisms/Navbar/Navbar/__snapshots__/Navbar.test.tsx.snap:23]: that's a bit strange

Comment (poteirard) [reply src/components/organisms/Navbar/Navbar/__snapshots__/Navbar.test.tsx.snap:23]: yes... provably a bug in jest-styled-components... It's a bit flacky with v5 of styled-components. I read they change many things internally

**Pull request merged**